shards/shards.go

```diff
@@ -639,7 +688,11 @@ func (s *shardedSearcher) getShards() []rankedShard {
 	for _, sh := range s.shards {
 		res = append(res, sh)
 	}
-	sort.Slice(res, func(i, j int) bool {
+	sort.SliceStable(res, func(i, j int) bool {
```

Comment:
perf: we don't need a stable sort since shards is a map (and as such res is shuffled each time)